README.md

```diff
@@ -125,6 +122,16 @@ Example for a "admin" group with higher limits.
     ...
 ```
 
+Sometimes you may want to specify that some APIs share the same flow control pool. In other words, flow control is performed on the entire set of APIs instead of a single specific API. Only the `zone` parameter needs to be used.
+
+```python
+    ...
+    config={
+        r"/user/\d+": [Rule(minute=200, zone="user-api")],
```

Comment:
I think zone should be the attribute at endpoint level not Rule level. Thoughts?